src/libcprover-rust/src/lib.rs

```diff
@@ -101,6 +101,22 @@ mod tests {
         assert_eq!(vect.len(), 2);
     }
 
+    // This test will capture a `system_exceptiont` from CBMC's end at the C++ shim that this
+    // library depends on, and it will be correctly translated into the Result type for Rust.
+    // This also validates that our type definition include of the base class for the exceptions
+    // works as we expect it to.
+    #[test]
+    fn it_translates_exceptions_to_errors() {
+        let client = cprover_api::new_api_session();
+
+        // The vector of string is supposed to contain a string denoting
+        // a filepath that is erroneous.
+        let vec: Vec<String> = vec!["/fkjsdlkjfisudifoj2309".to_owned()];
```

Comment:
I have not tried to break this, but what happens if I have a valid input file at `/fkjsdlkjfisudifoj2309`? I'm also curious if the `/` is OS specific to any degree?

Reply:
Well, that was my best attempt at making a random string + file path combo, without resorting to a more elaborate mechanism based on random strings or whatnot.

I'm personally happy with it failing if a user does have a *randomly* (?) named file locally present. If that happens, the user/developer can just have a look at the test, and see that we are looking for this file, inspect his filesystem and locate the file, and potentially alter the test/delete the file.